.github/workflows/build-boot-iso.yml

```diff
@@ -73,10 +73,10 @@ jobs:
           ref: ${{ needs.pr-info.outputs.sha }}
           fetch-depth: 0
 
-      - name: Build anaconda-iso-creator container image
+      - name: Build anaconda-iso-creator-webui container image
         run: |
           # set static tag to avoid complications when looking what tag is used
-          sudo make -f ./Makefile.am anaconda-iso-creator-build CI_TAG=$CONTAINER_TAG
+          sudo make -f ./Makefile.am anaconda-iso-creator-build CI_TAG=$CONTAINER_TAG ISO_CREATOR_NAME=$ISO_BUILD_CONTAINER_NAME
```

Comment:
This will not work, you need to change the source Dockerfile not name.

However, first try to just use the old one with the an adjustment.